packages/panzer/disc-fe/src/evaluators/Panzer_Integrator_BasisTimesVector.hpp

```diff
@@ -176,6 +176,51 @@ namespace panzer
       Integrator_BasisTimesVector(
         const Teuchos::ParameterList& p);
 
+      /**
+       *  \brief Descriptor Constructor.
+       *
+       *  Creates an `Evaluator` to evaluate the integral
+       *  \f[
+            Ma(x)b(x)\cdots\int\vec{s}(x)\cdot\vec{\phi}(x)\,dx,
+          \f]
+       *  where \f$ M \f$ is some constant, \f$ a(x) \f$, \f$ b(x) \f$, etc.,
+       *  are some fields that depend on position, \f$ \vec{s} \f$ is some
+       *  vector-valued function, and \f$ \vec{\phi} \f$ is some vector basis.
+       *
+       *  \param[in] evalStyle  An `enum` declaring the behavior of this
+       *                        `Evaluator`, which is to either:
+       *                        - compute and contribute (`CONTRIBUTES`), or
+       *                        - compute and store (`EVALUATES`).
+       *  \param[in] resTag    The tag of either the contributed or evaluated
+       *                        field, depending on `evalStyle`.
+       *  \param[in] valTag    The tag of the vector value being integrated
+       *                        (\f$ \vec{s} \f$).
+       *  \param[in] bd      The vector basisdescripotr that you'd like to use (\f$
+                                \vec{\phi} \f$).
+       *  \param[in] id      The integration descriptor that you'd like to use.
+       *  \param[in] multiplier    The scalar multiplier out in front of the
+       *                        integral you're computing (\f$ M \f$).  If not
+       *                        specified, this defaults to 1.
+       *  \param[in] multipleirs    A list of names of fields that are multipliers
+       *                            out in front of the integral you're computing
+       *                            (\f$ a(x) \f$, \f$ b(x) \f$, etc.).  If not
+       *                            specified, this defaults to an empty `vector`.
+       *
+       *  \throws std::invalid_argument If any of the inputs are invalid.
+       *  \throws std::logic_error      If the `basis` supplied is not a vector
+       *                                basis, or if it doesn't require
+       *                                orientations.
+       */
+      Integrator_BasisTimesVector(
+        const panzer::EvaluatorStyle&   evalStyle,
+        const PHX::Tag<typename EvalT::ScalarT> &       resTag,
+        const PHX::Tag<typename EvalT::ScalarT> &       valTag,
```

Comment:
Is there a reason the ctor uses a concrete Tag class? Could these be changed to the base class PHX::FieldTag to eliminate the template parameter? Not a big deal since everyone uses the concrete class, but might give us a little more flexibility in the future.

Reply:
Will do